driver/main.c

```diff
@@ -177,7 +181,8 @@ TRACEPOINT_PROBE(signal_deliver_probe, int sig, struct siginfo *info, struct k_s
 
 /* tracepoints `page_fault_user/kernel` don't exist on some architectures.*/
 #ifdef CAPTURE_PAGE_FAULTS
-TRACEPOINT_PROBE(page_fault_probe, unsigned long address, struct pt_regs *regs, unsigned long error_code);
+TRACEPOINT_PROBE(page_fault_user_probe, unsigned long address, struct pt_regs *regs, unsigned long error_code);
```

Comment:
Split page_fault_probe into `page_fault_user_probe` and `page_fault_kern_probe` like we do in bpf and modern bpf.

Reply:
It is needed to be able to check correct tracepoint.